api/proto/disperser/disperser.proto

```diff
@@ -150,8 +150,8 @@ enum BlobStatus {
 	// for at least one quorum.
 	INSUFFICIENT_SIGNATURES = 5;
 
-	// CONFIRMING means that the blob has been dispersed to DA nodes and it's waiting for the confirmation onchain
-	CONFIRMING = 6;
+	// DISPERSING means that the blob is currently being dispersed to DA Nodes and being confirmed onchain
```

Comment:
It's fine to move it but we should not change the values of the enum's.

Reply:
I think it will be confusing and easier to make mistakes if you list enums that are not ordered by its value. I'll remove the grouping and update the comments

Reply:
I don't see there is confusion or easier to make mistake. You just need to put a comment at the top of the message indicating what's the next available id to use. Field num is just for encoding whereas grouping is for human reading which has higher value.

Reply:
How about instead of having the comment grouping inside the enum we could just have the comment explaining the grouping above the enum and keep the ordering of the enum.

Reply:
ok that sounds fine

Reply:
> since "getResponseStatus" categorize dispersing as "BlobStatus_PROCESSING", why do we need to edit the proto file? is the idea to show "BlobStatus_DISPERSING" later?

We use it internally, and yeah this should be shown eventually (could be bundled together in a release when other breaking changes are necessary)